docs/edgeql/functions/converters.rst

```diff
@@ -23,3 +23,24 @@ the functions below allow more options for such conversions.
 
         db> SELECT to_json('{"hello": "world"}')['hello'];
         {'world'}
+
+.. eql:function:: std::to_str(json: json, fmt: OPTIONAL str={}) -> str
+
+    :index: stringify dumps
+
+    Return string representation of the input value.
+
+    When converting :eql:type:`json`, this type converter can take an
```

Comment:
"this type converter" -> "this function"

Reply:
OK, fixed the repetition.